src/libraries/System.Runtime.Intrinsics/ref/System.Runtime.Intrinsics.cs

```diff
@@ -4138,6 +4138,23 @@ public new abstract partial class Arm64 : System.Runtime.Intrinsics.Arm.AdvSimd.
             internal Arm64() { }
             public static new bool IsSupported { get { throw null; } }
         }
+        public static System.Numerics.Vector<sbyte> Abs(System.Numerics.Vector<sbyte> value) { throw null; }
+        public static System.Numerics.Vector<short> Abs(System.Numerics.Vector<short> value) { throw null; }
+        public static System.Numerics.Vector<int> Abs(System.Numerics.Vector<int> value) { throw null; }
+        public static System.Numerics.Vector<long> Abs(System.Numerics.Vector<long> value) { throw null; }
+        public static System.Numerics.Vector<float> Abs(System.Numerics.Vector<float> value) { throw null; }
+        public static System.Numerics.Vector<double> Abs(System.Numerics.Vector<double> value) { throw null; }
```

Comment:
The tooling also expects these to be in alphabetical order, based on the fully qualified type name.

So `byte, double, short, int, long, nint, sbyte, float, ushort, uint, ulong, nuint` (corresponding to `Byte, Double, Int16, Int32, Int64, IntPtr, SByte, Single, UInt16, UInt32, UInt64, UIntPtr`)